### DISCUSSION TITLE
Extract 3D geometry from a patch as fbx/obj/gltf

### DISCUSSION THREAD

**Discussion by jaalibandar** [cables-gl/cables_docs#128] [Feature Requests]
Hi, I was playing around with the ParametricSurface node(its superawesome) - however it would be even better if there was some way to extract the geometry from the patch. Cables is not a 3D modelling software - I know - but it would be really handy to be able to have a few basic boolean functions(for geometries) - similar to merge geometry maybe? - and the ability to export all geometries present in the patch in some format

**Comment by franckaubry** [comment]
As we heard, this sounds like it can be a very difficult task.
One of the use cases that I thought about, is if you make geo in Cables with some kind of geomerging and loads of editing, it would be awesome to be able to "bake it" somehow so as to be able to use it as instancing.

**Reply by jaalibandar** [reply]
well i was asking mostly from a reusability perspective or a possibility to bring things out of cables in the future to use in other applications maybe - a roundabout way of doing it seems to be the render2texture op - and maybe export images for geometries and reuse those images to get those geometries - doesnt exactly work as expected currently and its still in dev but seems like a workable solution atm ...and its a pretty awesome op as well
